traffic_router/core/src/test/java/org/apache/traffic_control/traffic_router/core/router/StatTrackerTest.java

```diff
@@ -38,6 +38,6 @@ public void testIncTally() {
         track.setRouteType(StatTracker.Track.RouteType.DNS, "blah");
         track.setResult(StatTracker.Track.ResultType.CZ);
         tracker.saveTrack(track);
-        assertEquals("expected czCount to be max integer value but got " + tallies.getCzCount(), Integer.MAX_VALUE, tallies.getCzCount());
+        assertEquals("expected czCount to be max integer value but got " + tallies.getCzCount(), Long.MAX_VALUE, tallies.getCzCount());
```

Comment:
Assertion message should be updated too 🙂

Reply:
Oops, forgot to update that. Fixed now :)